src/message/action.js

```diff
@@ -879,8 +879,10 @@ function QueueReset() {
  * @property {String} Interface Interface
  * @augments Action
  */
-function QueueRemove() {
+function QueueRemove(queue, asteriskInterface) {
```

Comment:
It depends if you want to follow the official Asterisk parameter order. It does seem strange that just for the pause function, Asterisk switches the queue/interface parameter order though. 
Happy to go either way. 
(I do see where you're coming from setting the variables outside the action call :-) )